CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+Please keep in mind that your "Contribution(s)" are submitted under
+the [Apache 2.0 License](LICENSE).
+
+#### Updating j2objc-gradle
+The version of j2objc-gradle in this repo is not automatically kept up
+to date with master of https://github.com/j2objc-contrib/j2objc-gradle.
+
+As a submodule, you can update j2objc-gradle by doing:
+
+```shell
+# (start with a clean git working directory)
+# (from your local repo for this project)
+git checkout -b update-plugin
+pushd j2objc-gradle
+git pull
+popd
+git add .
+git commit -m 'Updating j2objc-gradle to HEAD'
+# (submit the update-plugin branch as a PR to the repo)
+```
+
+Make sure this PR is seperate from any other work.  If a library you want
+to build depends on the update, work in a new branch parented to the commit
+above.
```

Comment:
Clarify whether people should be updating to the plugin HEAD or only release versions of the plugin?